src/components/workspace-utils.js

```diff
@@ -97,7 +97,11 @@ export const withWorkspaces = (WrappedComponent) => {
 export const WorkspaceSelector = ({ workspaces, value, onChange, id, 'aria-label': ariaLabel, ...props }) => {
   const options = _.flow(
     _.sortBy((ws) => ws.workspace.name.toLowerCase()),
-    _.map(({ workspace: { workspaceId, name } }) => ({ value: workspaceId, label: name }))
+    _.map((workspace) => ({
+      value: workspace.workspace.workspaceId,
+      label: workspace.workspace.name,
+      workspace: workspace.workspace,
```

Comment:
What are the specific things you need out of the workspace?

Reply:
Just `cloudPlatform` and `bucketName` - I originally only passed those two variables along but was advised to just throw the whole thing in there - that conversation seems to have disappeared with the code change though.

Reply:
But here you pass `workspace.workspace`, which is contradictory to the code at https://github.com/DataBiosphere/terra-ui/pull/4019/files#r1267156537, which it seems you are expected `workspace` itself.

I would be tempted to just pass the things you actually need because it's so much clearer what is going on.

Reply:
You're right, this is very confusing.  I'll change it back to only passing what I need!